bigtable/tests/unit/test_table.py

```diff
@@ -1231,6 +1246,24 @@ def __next__(self):  # pragma: NO COVER Py3k
         return self.next()
 
 
+class _MockFailureIterator(object):
+
+    def next(self):
+        class DeadlineExceeded(grpc.RpcError, grpc.Call):
+            """ErrorUnavailable exception"""
+
+            def code(self):
+                return grpc.StatusCode.DEADLINE_EXCEEDED
+
+            def details(self):
+                return "Failed to read from server"
+
+        raise DeadlineExceeded()
+
+    def __next__(self):  # pragma: NO COVER Py3k
```

Comment:
just do `__next__ = next`

Reply:
@jonparrott This change is not working on python 2.7.

Reply:
err, rather, the other way around.